README.md

```diff
@@ -80,14 +83,18 @@ Release "Purely Easy Wahoo"
 
 Save this release? [y/N]:
 ```
+The release plan will be saved inside the `<workspace director>/release.yaml` file.
```

Comment:
```suggestion
The release plan will be saved inside the `<workspace directory>/release.yaml` file.
```

Reply:
done.
I have also changed the directory path to `<workspace directory/.sebex/release.yaml`